amgut/templates/no_auth_help_request.html

```diff
@@ -0,0 +1,40 @@
+{% extends no_auth_sitebase.html %}
```

Comment:
It's the base template for the site when the user is logged out, so nothing will call it directly.